### PR TITLE
Fully Import P2RA distributions

### PR DESCRIPTION
**Pull request by jeffkaufman**
Finish the work started in #7 to support allowing users to directly select the distributions we produce with P2RA.

The key changes are:
* Make the CDF of RAi(1%) estimates imported in #6 available as ww_rai1pct
* Pull the inverse transform sampling that we previously used only for coverage out into a function.
* Draw from the CDF of RAi(1%) estimates on a per-simulation basis.
* Make a mode where we simulate using RAi(1%) instead of estimating (a) the fraction of sick people and (b) the relative abundance contribution of an individual sick person.

You can play with this live at
https://data.securebio.org/simulator/index-ww-dist.html

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -534,6 +531,22 @@ <h1>Sampling and Sequencing Simulator</h1>
   return adjusted_values;
 }
 
+function inverse_transform_sample(cdf) {
```
Comment (dp-rice) [inline html/index.html:534]: This looks correct.

While thinking about the problem I came across this [faster algorithm](https://en.wikipedia.org/wiki/Alias_method) for generating discrete random variables. No reason to switch to it unless this somehow becomes the bottleneck step.

File: html/index.html [hunk 2]
```diff
@@ -647,38 +666,59 @@ <h1>Sampling and Sequencing Simulator</h1>
         const n_sick = poisson(n_sample_population * individual_probability_sick);
         site_infos[site].sample_sick += n_sick;
         site_infos[site].sample_total += n_sample_population;
+
+        // Compute weekly incidence from daily incidence
+        let weekly_incidence = 0;
+        effective_incidence = daily_incidence;
+        for (let i = 0; i < 7; i++) {
+           weekly_incidence += effective_incidence;
+           effective_incidence /= growth_factor;
```
Comment (dp-rice) [inline html/index.html:675]: I don't understand this block. Why isn't the weekly incidence the sum of the daily incidence over the week?

**Review by dp-rice — APPROVED** [review]
I don't see any problems here, but I think the complexity is really starting to push the bounds of what's reasonable to do in this setting with all these hand-rolled sampling algorithms. I'm having some difficulty following all the logic. Is there any way in a future PR that we could move the simulation logic to, e.g., numpy, so we don't have to debug all these loops? (It would also be faster so we wouldn't have to do so much control flow for the sake of efficiency.)

File: html/index.html (hunk 2 above)
Comment (jeffkaufman) [reply html/index.html:675]: > Why isn't the weekly incidence the sum of the daily incidence over the week?

That is what this block is calculating, yes.  Weekly incidence is the sum of how many people became sick (daily incidence) on each of the previous seven days.

**Comment by jeffkaufman** [comment]
> Is there any way in a future PR that we could move the simulation logic to, e.g., numpy, so we don't have to debug all these loops?

We can't move it to numpy in particular because this is running in the user's browser (and so needs to be JS).

It's possible that there's some JS equivalent of numpy that we could use, though, or just a way we could rewrite this to be easier to understand.  I'll have a go at the latter.

**Comment by jeffkaufman** [comment]
>  I'll have a go at the latter.

Done as #11

**Pull request merged**